kong/plugins/log-serializers/basic.lua

```diff
@@ -12,11 +12,12 @@ function _M.serialize(ngx)
       consumer_id = ngx.ctx.authenticated_credential.consumer_id
     }
   end
-  
+
   return {
     request = {
-      uri = ngx.var.request_uri,
-      request_uri = ngx.var.scheme .. "://" .. ngx.var.host .. ":" .. ngx.var.server_port .. ngx.var.request_uri,
+      request_uri = ngx.var.request_uri,
+      upstream_uri = ngx.var.uri,
```

Comment:
Since 0.11.0, the upstream URI is determined via `ngx.var.upstream_uri`, and not set via `ngx.req.set_uri()` anymore, which means that this value need to be updated from `ngx.var.uri` to `ngx.var.upstream_uri`. Could you take care of that?